examples/module1/lesson1/solver/Operations.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button } from './Button.tsx';
+import operations from './operations.ts';
+
+type Props = {
+  setResult: (result: number | string) => void
+  x: number
+  y: number
+}
+
+export const Operations = ({setResult, x, y}:Props) => {
+  return (
+    <div className="grid grid-cols-4 gap-x-4 my-4">
+      {operations.map(({ execution, operator, name }) =>
+        name === 'divide' ? (
```

Comment:
Try to avoid low-level conditions like "if divide" - how to handle this error outside of this component?